tests/test.js

```diff
@@ -42,3 +42,28 @@ test('build next app with manifest and service worker', async () => {
   const str = await read(join(cwd, '.next/main.js'), 'utf8');
   expect(str).toEqual(expect.stringContaining('serviceWorker'));
 });
+
+test('pre-caching includes static assets and bundles', async () => {
+  const nextConf = forceProd(
+    withOffline(
+      withManifest({
+        manifest: {
+          name: 'next-app',
+        },
+      }),
+    ),
+  );
+
+  await nextBuild.default(cwd, nextConf);
+  const str = await read(join(cwd, '.next/service-worker.js'), 'utf8');
+
+  // Check that various bundles are getting entered into pre-cache manifest
+  expect(str).toEqual(expect.stringContaining('/pages/_app.js'));
+  expect(str).toEqual(expect.stringContaining('_next/static/chunks/commons.'));
+
+  // Check that static asset copying via glob pattern is working as expected
+  expect(str).toEqual(expect.stringContaining('_next/static/image.jpg'));
+
+  // TODO: determine whether we should be excluding this file. Not sure if it should be pre-cached
```

Comment:
@hanford any thoughts? It appears this was previously being pre-cached but I just want to confirm this is desired. Admittedly, I'm just entering the PWA world so I'm not sure if this is a best practice or what.